modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc

```diff
@@ -74,29 +75,17 @@ Check if the package `docker-compose` is installed in addition to docker:
 which docker-compose
 ----
 
-If `docker-compose` is installed, you'll be informed. If not, you may get no output at all or a message that it couldn't be found. In that case you need to install `docker-compose` first. On most Linux distributions, you can simply use the package manager to do so. Note that in many cases, this will install a 1.x version (python-based) while a 2.x version (go-based) is preferred. To check which version would be installed with the package manager, simply type:
+If `docker-compose` is installed, you'll be informed. If not, you may get no output at all or a message that it couldn't be found. In that case you need to install `docker-compose` first. On most Linux distributions, you can simply use the package manager to do so.
 
-[source,bash]
-----
-sudo apt-cache policy docker-compose
-----
-
-If the output shows a version like 2.x, you can use the package manager to install docker-compose, like with the following command on Debian-based distributions:
-
-[source,bash]
-----
-sudo apt install docker-compose
-----
-
-If the output shows a version like 1.25.0-1, follow the {docker-compose-install-url}[Install Docker Compose] guide to install a 2.x version.
+Note that in many cases, this will install a 1.x version (python-based) only supported until June 2023. Version 2.x (go-based) is available from Docker. Follow the {docker-compose-install-url}[Install Docker Compose] guide to install a 2.x version. Familiarize yourself with the {docker-compose-evolution}[differences between versions] particularly with regard to syntax changes. Most of our examples are still based on Compose-V1.
```

Comment:
```suggestion
Note that in many cases, this will install a 1.x version (python-based) only supported until June 2023. Version 2.x (go-based) is available from Docker. Follow the {docker-compose-install-url}[Install Docker Compose] guide to install a 2.x version. Familiarize yourself with the {docker-compose-evolution-url}[differences between versions] particularly with regard to syntax changes. Most of our examples are still based on Compose-V1.
```